wolfcrypt/src/random.c

```diff
@@ -838,6 +838,11 @@ static int _InitRng(WC_RNG* rng, byte* nonce, word32 nonceSz,
     rng->status = DRBG_NOT_INIT;
 #endif
 
+#if defined (WOLF_CRYPTO_CB) && defined(WOLFSSL_MAXQ108x)
```

Comment:
Loose space `defined(WOLF_CRYPTO_CB)`. Hard coding when `WOLFSSL_MAXQ108x` is defined is a bit of a hack. Technically you can set all this via TLS using `wolfSSL_SetDevId` at run-time. I saw another place this was hard coded. Also is it only the WOLFSSL_MAXQ108x the part that supports RNG?

Reply:
I think they do this in case a user wanted to use wolfcrypt without wolfssl. Maybe I can figure out something better.  I should test if this is even required anymore. I've asked if 1065 also supports RNG.

Reply:
Removed.  Its already set to the correct devid via changes I made.